Patches/Mechanoid Bench 3 JGH/Hediff_MB3.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+
+    <Operation Class="PatchOperationFindMod">
+        <mods>
+            <li>Mechanoid Bench 3</li>
+        </mods>
+        <match Class="PatchOperationSequence">
+
+            <operations>
+
+                <!-- ===== Mecha Bionic Lens ===== -->
+                <li Class="PatchOperationReplace">
+                    <xpath>/Defs/HediffDef[defName="JGH_MechaBionicEye"]/stages/li/statOffsets</xpath>
+                    <value>
+                        <statOffsets>
+                            <AimingAccuracy>0.2</AimingAccuracy>
+                            <NightVisionEfficiency_Implant>0.4</NightVisionEfficiency_Implant>
+                        </statOffsets>
+                    </value>
+                </li>
+
+                <!-- ===== Mecha Bionic Ear ===== -->
+                <li Class="PatchOperationReplace">
+                    <xpath>/Defs/HediffDef[defName="JGH_MechaBionicEar"]/stages/li/statOffsets/MeleeDodgeChance</xpath>
+                    <value>
+                        <MeleeDodgeChance>0.05</MeleeDodgeChance>
+                    </value>
+                </li>
+                
+                <!-- ===== Mecha Bionic Spine ===== -->
+                <li Class="PatchOperationAdd">
+                    <xpath>/Defs/HediffDef[defName="JGH_MechaBionicSpine"]/stages/li/statOffsets</xpath>
+                    <value>
+                        <CarryWeight>20</CarryWeight>
+                    </value>
+                </li>
+
+                <!-- ===== Mecha Bionic Arm ===== -->

```

Comment:
looks kinda weak

Reply:
well kinda, I do think it's similar to bionic arm. I might change it to be onpar with power arm in terms of melee stats, as for the mod statOffsets it was reduced Aiming Delay

Reply:
haven't touched them at all so it still remains the same